package.json

```diff
@@ -18,13 +18,23 @@
   ],
   "author": "Appcelerator, Inc. <npmjs@appcelerator.com>",
   "scripts": {
-    "test": "npm run ios-sanity-check && grunt",
+    "test": "npm run ios-sanity-check && npm run lint",
     "android": "./build/scons cleanbuild android",
     "android-test": "./build/scons clean android && ./build/scons build android && ./build/scons package -s --no-docs android && ./build/scons install -s && ti sdk select $npm_package_version && ./build/scons test -s android",
     "ios": "./build/scons cleanbuild ios",
     "ios-sanity-check": "./build/scons check-ios-toplevel",
     "ios-test": "./build/scons clean ios && ./build/scons build ios && ./build/scons package -s --no-docs ios && ./build/scons install -s && ti sdk select $npm_package_version && ./build/scons test -s ios",
-    "commit": "git-cz"
+    "commit": "git-cz",
+    "lint": "npm-run-all --parallel lint:**",
+    "lint:android": "./build/scons format --android",
+    "lint:docs": "tdoc-validate ./apidoc",
+    "lint:ios": "./build/scons format --ios",
+    "lint:js": "eslint .",
+    "format": "npm-run-all --parallel format:**",
+    "format:android": "./build/scons format --android --fix",
+    "format:ios": "./build/scons format --ios --fix",
+    "format:js": "npm run lint:js -- --fix",
+    "unit-tests": "nyc --exclude **/cli/tests/test-*.js mocha **/cli/tests/test-*.js"
```

Comment:
so in keeping with this theme, I would suggest we name this "test:unit" and migrate "android-test" and "ios-test" to be "test:android" and "test:ios" respectively.

Reply:
or maybe even "test:cli"? "test:js" hopefully align the naming across the various actions.

Reply:
Added the following `test:android`, `test:cli`, `test:ios`.

I didn't point `npm test` to run these though as I think that is probably unwanted. I also wasn't sure on whether to have `cli` or `js` in the naming, I went with the former as while it isn't inline with the format/lint steps, it is clearer as to what the task does.